app/models/form/sales/pages/handover_date_check.rb

```diff
@@ -1,7 +1,7 @@
 class Form::Sales::Pages::HandoverDateCheck < ::Form::Page
   def initialize(id, hsh, subsection)
     super
-    @depends_on = [{ "hodate_3_years_or_more_saledate?" => true }]
+    @depends_on = [{ "hodate_3_years_or_more_exdate?" => true }]
```

Comment:
things related to this are fixes from another PR that wasn't doing what I think it should have been